docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-en.md

```diff
@@ -0,0 +1,105 @@
+
+|Dashboard version|Nebula Graph version|
+|:---|:---|
+|{{ dashboard-en.release }}|2.x|
+
+## tar-based Dashboard
+
+### Prerequisites
+
+Before you deploy tar-based Dashboard, you must do a check of these:
+
+- The environment of [MySql](https://www.mysql.com/) is ready and a database named as `dashboard` is created.
+- Before the installation starts, the following ports are not occupied.
+  
+  | Port | Description |
+  | ---- | ---- |
+  | 7005 | The port through which Dashboard provides the web service. |
+  | 8090 | The port through which nebula-http-gateway provides the HTTP service. |
+  | 9090 | The port of the prometheus service. |
+  | 9200 | The port of the nebula-stats-exporter service. |
+
+- The license is ready.
+
+  !!! enterpriseonly
+
+        The license is only available in the Enterprise Edition. To obtain the license, send an email to inquiry@vesoft.com.
+
+### Install and start
+
+1. Select and download the tar package according to your needs. It is recommended to select the latest version.
+
+  !!! enterpriseonly
+
+        For features of Dashboard Enterprise Edition, see [Pricing](https://nebula-graph.com.cn/pricing/).
+
+2. Use `tar -xzvf` to decompress the tar package.
+
+   ```bash
+   $ tar -xzvf nebula-graph-dashboard-<version>.tar.gz
+   ```
+
+3. Use `vim config/config.yaml` to modify the configuration.
+
+   ```bash
+   # Information about the database
+   database:
+     dialect: mysql # The type of database used, which currently only supports MySql
+     host: 192.168.8.157 # The IP address of the connected MySql database
+     port: 3306 # The port of the connected MySql database
+     username: root # The username to log in MySql
+     password: nebula # The password to log in MySql
+     name: dashboard # The name of the corresponding database
+     autoMigrate: true # Auto database tables creation, the default value of which is true
+   # Information about the exporter port
+   exporter:
+     nodePort: 9100 # The port of the node-exporter service
+     nebulaPort: 9200 # The port of the nebula-stats-exporter service
+   # Information of services
+   proxy:
+     gateway:
+       target: "localhost:8090" # The IP address and port of the gateway service
+     prometheus:
+       target: "localhost:9090" # The IP address and port of the prometheus service
+   ```
+
+4. Copy the license to the `nebula-dashboard` directory.
+
+   ```bash
+   $ cp -r <license> <dashboard_path>
+   ```
+
+   For example:
+   ```bash
+   $ cp -r nebula.license /usr/local/nebula-dashboard
+   ```
+
+  !!! enterpriseonly
+
+        The license is only available in the Enterprise Edition. To obtain the license, send an email to inquiry@vesoft.com.
+
+5. Start Dashboard.
+
+   ```bash
+   $ bash ./scripts/start.sh
+   ```
+
+### Stop service
+
+```bash
+$ bash ./scripts/stop.sh
+```
+
+## Next to do
+
+After Dashboard is successfully started, you can enter `http://<ip_address>:7005` in the address bar of the browser.
```

Comment:
in the address bar of a browser

if using "the browser" please specify which browser to use.